client/src/components/ProductOverview/ProductOverview.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import axios from 'axios';
+
+import ImageGallery from './ImageGallery';
+import ProductInformation from './ProductInformation';
+import StyleSelector from './StyleSelector';
+import AddToCart from './AddToCart';
+import OverviewAndShare from './OverviewAndShare';
+
+class ProductOverview extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isFetching: true,
+      expanded: false,
+      product: {},
+      productStyles: {},
+      productReviews: {},
+      productRatings: {},
+      productStarRatings: 3.7, // Hardcoded for now to play around with it
+    };
+  }
+
+  componentDidMount() {
+    Promise.all([
+      // For now, grab all products from the API and set the current product as the 1st one (40344)
+      ProductOverview.getProductList(),
+      // For now, grab product details only for the 1st product
+      ProductOverview.getProductStyles(40344),
+      ProductOverview.getProductReviews(40344),
+      ProductOverview.getProductRatings(40344),
+    ])
+      .then((results) => {
+        // console.log(results);
+        this.setState({
+          product: results[0].data[0],
+          productStyles: results[1].data,
+          productReviews: results[2].data,
+          productRatings: results[3].data,
+          isFetching: false,
+        });
+      })
+      .catch((error) => { throw new Error(`Error in fetching from server: ${error.message}`); });
+  }
+
+  static getProductList() {
```

Comment:
Never used static before, can you describe what it does?

Reply:
it was to appease the airbnb linter -  if any of your class methods don't use "this", then that most likely implies that the method isn't meant to be shared across instances of the class (i.e. is only used once). in that case you should use "static" to denote your method is only used once for that one instance of the class.

Reply:
makes sense b/c i will only ever have one Product Overview component and it's not a true class with diff instances